src/site/content/en/blog/eventing-deepdive/index.md

```diff
@@ -0,0 +1,572 @@
+
+What other element/event combinations allow you to prevent the default action? I cannot possibly
+list them all, and sometimes you have to just experiment to see. But briefly, here are a few:
+
+- `<form>` element + "submit" event: `preventDefault()` for this combination will prevent a form from submitting. This is useful if you want to perform validation and should something fail, you can
+  conditionally call preventDefault to stop the form from submitting.
+
+- `<a>` element + "click" event: `preventDefault()` for this combination prevents the browser from
+  navigating to the URL specified in the `<a>` element's href attribute.
+
+- `document` + "mousewheel" event: `preventDefault()` for this combination prevents page scrolling
+  with the mousewheel (scrolling with keyboard would still work though). <br>
+  <small>^ Note that this particular combination requires calling `addEventListener()` with `{ passive: false }`</small>.
+
+- `document` + "keydown" event: `preventDefault()` for this combination is lethal. It renders the page largely useless, preventing keyboard scrolling, tabbing, and keyboard highlighting.
+
+- `document` + "mousedown" event: `preventDefault()` for this combination will prevent text
+  highlighting with the mouse and any other "default" action that one would invoke with a mouse
+  down.
+
+- `<input>` element + "keypress" event: `preventDefault()` for this combination will prevent characters typed by the user from reaching the input element (but don't do this; there is rarely, if ever, a valid reason for it).
+
+- `document` + "contextmenu" event: `preventDefault()` for this combination prevents the native
+  browser context menu from appearing when a user right-clicks or long-presses (or any other way in which a context menu might appear).
+
+This is not an exhaustive list by any means, but hopefully it gives you a good idea of how
+`preventDefault()` can be used.
+
+### A fun practical joke?
+
+What happens if you `stopPropagation()` _and_ `preventDefault()` in the capturing phase, starting at
+the document? Hilarity ensues! The following code snippet will render any web page just about
+completely useless:
+
+```js
+function preventEverything(e) {
+  e.preventDefault();
+  e.stopPropagation();
+  e.stopImmediatePropagation();
+}
+
+document.addEventListener('click', preventEverything, true);
+document.addEventListener('keydown', preventEverything, true);
+document.addEventListener('mousedown', preventEverything, true);
+document.addEventListener('contextmenu', preventEverything, true);
+document.addEventListener('mousewheel', preventEverything, { capture: true, passive: false });
+```
+
+I don't really know why you'd ever want to do this (except maybe to play a joke on someone), but it
+is useful to think about what's happening here, and realize why it creates the situation it does.
+
+All events originate at the document, so in this snippet, we're stopping, dead in their tracks, all
```

Comment:
here's a curveball: what does it do to Shadow DOM?
(I'm not super fussed about adding anything here, because this is a general intro, but it could be interesting.)

Reply:
Ooo! Great point.  I'm honestly not sure.  I will do some experimenting and maybe update this later if I find anything interesting, but can we agree that it's not necessary to cover this in order to publish V1?